core/src/test/resources/project.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+project.basedir=${basedir}
```

Comment:
I created 4 other these properties files, one for each project. Its sole purpose is to accurately get the project's base directory at test time, regardless of the directory Maven is invoked. The test in each project is almost identical, except for the path at which it expects the `deps.txt` file. The test is about as simple as it gets and just verifies the file is generated.

In the future, if we forget to generate `deps.txt` this test should fail. However, it won't catch new `deps.txt` files that we may want to create going forward, so we would have to copy this test as needed. Unless we want one for every child project, in which case I could refactor this some.

Reply:
This still isn't quite right. the resource filtering isn't working the way I intended.